### Import Libraries

In [1]:
import pandas as pd
from scipy import stats

### Import Data

In [2]:
cutlets=pd.read_csv("cutlets.csv")
cutlets.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

### Hypothesis Test

In [3]:
stats.ttest_ind(cutlets['Unit A'], cutlets['Unit B'], alternative='two-sided')

Ttest_indResult(statistic=0.7228688704678063, pvalue=0.4722394724599501)

### Inference

HYPOTHESIS
**********
Ho : There is no significant difference in diameter of cutlets
Ha : There is significant difference in diameter of cutlets

SOLUTION
********
alpha : 0.05
pvalue found : 0.47
pvalue is greater than alpha, Hence Ho is failed to reject

CONCLUSION
**********
There is no significant difference in diameter of cutlets.

### End !!!